src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using CalculatorApp;
+using CalculatorApp.ViewModel;
+
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using System.ComponentModel;
+
+namespace CalculatorApp
+{
+    public sealed partial class KeyGraphFeaturesPanel : System.ComponentModel.INotifyPropertyChanged
+    {
+        public KeyGraphFeaturesPanel()
+        {
+            InitializeComponent();
+            this.Loaded += KeyGraphFeaturesPanel_Loaded;
+        }
+
+        public event PropertyChangedEventHandler PropertyChanged;
+        internal void RaisePropertyChanged(string p)
+        {
+#if !UNIT_TESTS
+            PropertyChanged(this, new PropertyChangedEventArgs(p));
```

Comment:
check null reference before raising up this event

Reply:
Used ?.Invoke() instead to avoid null reference issue.